src/components/action/validation/validationActionComponents/ValidationSettings.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Collapse } from 'reactstrap';
+
+import CheckBox from '../../checkBox/CheckBox';
+import './ValidationSettings.css';
+import ValidationProfile from './ValidationProfile';
+import FixMetadata from './FixMetadata';
+
+class ValidationSettings extends Component {
+    fileInput = null;
+
+    renderStopValidationTitle = () => {
+        const { amountOfFailedChecks, selectAmountOfFailedChecks } = this.props;
+        return (
+            <Fragment>
+                <span>Stop validation after</span>
+                <input
+                    defaultValue={amountOfFailedChecks}
+                    onChange={e => selectAmountOfFailedChecks(e.target.value)}
+                    className="failed-checks-input"
+                    type="number"
+                />
+                <span>failed checks</span>
+            </Fragment>
+        );
+    };
+
+    render() {
+        const {
+            profile,
+            profilesValues,
+            profileFilePath,
+            getValidationProfileValue,
+            setCustomProfilePath,
+            isFixMetadata,
+            selectFixMetadata,
+            fixedFilePrefix,
+            selectFixedFilePrefix,
+            reportPassedRules,
+            toggleReportPassedRules,
+            stopValidationThreshold,
+            toggleStopValidationThreshold
+        } = this.props;
+
+        return (
+            <div>
+                <ValidationProfile
+                    porfile={profile}
+                    profilesValues={profilesValues}
+                    setCustomProfilePath={setCustomProfilePath}
+                    profileFilePath={profileFilePath}
+                    getValidationProfileValue={getValidationProfileValue}
+                />
+                <div>
```

Comment:
Can we get rid of this `<div>` wrapper? It looks a bit redundant... Same for 'validation-checks-fail-container' below. If needed add className prop to checkbox itself.